plugins/inputs/t128_lte/t128_lte.go

```diff
@@ -256,28 +253,24 @@ func matchNode(nodeName string, content []byte) (string, error) {
 	return "", fmt.Errorf("No matching node names are available in the config %s", content)
 }
 
-func (plugin *T128Lte) getDeviceInterface(name string, content []byte) (lteDeviceInterface, error) {
+func (plugin *T128Lte) getDeviceInterface(name string, content []byte) ([]lteDeviceInterface, error) {
 	var lte128Config t128Config
 	err := json.Unmarshal(content, &lte128Config)
 	if err != nil {
-		return lteDeviceInterface{}, err
+		return []lteDeviceInterface{}, err
 	}
-	var deviceInterfaceContent lteDeviceInterface
+	var deviceInterfaceContent []lteDeviceInterface
 	for _, router := range lte128Config.Config.Authority.Router {
 		for _, node := range router.Node {
```

Comment:
This should be working on the piece of the config identified in `matchNode`. As written, it will sample from both nodes in an HA pair instead of only the local one.